hw/syn/tools/dc/run-syn.tcl

```diff
@@ -84,7 +84,7 @@ check_design                                              > "${REPDIR}/check.rpt
 set_verification_top
 
 write_file -format ddc -hierarchy -output "${DDCDIR}/elab.ddc"
-write_file -format verilog -hierarchy -output "${DDCDIR}/elab.v"
+#write_file -format verilog -hierarchy -output "${DDCDIR}/elab.v"
```

Comment:
we shouldn't be writing out the verilog netlist here yet, since it is unmapped.

Reply:
@cdgori FYI

Reply:
I saw this change - was this the root cause of all of those warnings?!  (I saw that they went away in most recent regression)

Reply:
yes, since there are unmapped elements in the netlist. writing out DDC is fine at this stage, but a verilog netlist might not be correctly read back in.